### ISSUE TITLE
lotta broken links in README.md

### ISSUE THREAD

**Issue by Narvey** [xs-admin/Excess#4]
You have a lot of links that look like just placeholders in your README.md.
http://doc.xs, http://doc.extensions, http://roslyn, etc.

**Comment by xs-admin** [comment]
yes, unfortunately the project is practically abandoned at the moment, but
that could change with contributors.

**Comment by ghost** [comment]
yeah so I'd like to add static abstract functionality for my meta programming project and I would like to read into Excess. (shouldn't be too hard actually but having seen the new c# 8 language features I got fucking enough of that crap team...) Do you have the documentation lying around somewhere?

**Comment by xkp** [comment]
> yeah so I'd like to add static abstract functionality for my meta programming project and I would like to read into Excess. (shouldn't be too hard actually but having seen the new c# 8 language features I got fucking enough of that crap team...) Do you have the documentation lying around somewhere?

Hi, not much documentation but a lot of samples, if you give me some examples of what you want to do I can point you to the right place. Or let you know it is not doable :)
